buildslave/create.sh

```diff
@@ -10,6 +10,6 @@ vagrant box update buildslavebox
 vagrant up buildslavebox
 vagrant package buildslavebox --output buildslave.box
 vagrant destroy -f buildslavebox
-vagrant box remove -f buildslavebox
+vagrant box remove -f buildslavebox || echo "Warning: Couldn't remove previous basebox"
 vagrant box add buildslavebox buildslave.box
```

Comment:
Isn't the `box add` going to fail if failing the previous basebox fails? Or is this for the cases when no previous basebox exists?

Reply:
No, my remove was failing (likely because of no previous box) and this change fixed the problem. The whole script is running with set -e, so I think it's okay for remove to fail if add succeeds.